cypress.json

```diff
@@ -1 +1,3 @@
-{}
+{
+  "ignoreTestFiles": "*.js"
+}
```

Comment:
Config to show only features files in cypress test runner UI.

e.g : After config : 
![image](https://user-images.githubusercontent.com/2706268/62413214-d7634680-b62a-11e9-96a9-9ae08a6453f8.png)

Without this config  : 
![image](https://user-images.githubusercontent.com/2706268/62413226-f06bf780-b62a-11e9-8253-a2f0c5665615.png)